src/test/java/seedu/duke/task/TaskListTest.java

```diff
@@ -26,9 +26,9 @@ public void isEmpty_nonEmptyList_false() {
     public void testDeleteTask() {
         try {
             TaskList taskList = new TaskList();
-            taskList.addTask(new Task("Do CS2113T", "sat", "Level 1 increment"));
-            taskList.addTask(new Task("Do CS2113T", "sat", "Level 2 increment"));
-            taskList.addTask(new Task("Do CS2113T", "sat", "Level 3 increment"));
+            taskList.addTask(new Task("Do CS2113T", "SAT", "Level 1 increment"));
+            taskList.addTask(new Task("Do CS2113T", "SAT", "Level 2 increment"));
+            taskList.addTask(new Task("Do CS2113T", "SAT", "Level 3 increment"));

```

Comment:
Actually no need to capitalize.

https://github.com/AY2122S1-CS2113T-W11-3/tp/blob/b62548f5c294feb04be37e41e224306e7338fe0d/src/main/java/seedu/duke/parser/DayOfTheWeek.java#L16

I'm gonna improve the `DayOfTheWeek` enum in future commits to make it more flexible.